.vscode/launch.json

```diff
@@ -7,6 +7,7 @@
             "name": "Launch Extension",
             "runtimeExecutable": "${execPath}",
             "args": [
+                "--disable-extensions",
```

Comment:
Looks like a left-over...

Reply:
I haven't done vscode extension development before, but it seems useful to only have the extension under development being loaded in debug. A google search turned up this way of doing it.
Would you like me to remove it?

Reply:
@mike-lischke can you resolve this or ask me to remove it.

Reply:
I'll remove it.